src/app/products/floating-rates/edit-floating-rate/edit-floating-rate.component.ts

```diff
@@ -55,13 +56,15 @@ export class EditFloatingRateComponent implements OnInit {
    * @param {ActivatedRoute} route Activated Route.
    * @param {DatePipe} datePipe Date Pipe.
    * @param {MatDialog} dialog Dialog reference.
+   * @param {SettingsService} settingsService Settings Service.
    */
   constructor(private router: Router,
-              private formBuilder: FormBuilder,
-              private productsService: ProductsService,
-              private route: ActivatedRoute,
-              private datePipe: DatePipe,
-              private dialog: MatDialog) {
+    private formBuilder: FormBuilder,
```

Comment:
Consider vertical alignment as mentioned above. This should be applicable for all the components you have modified.